pyinfraformat/core/coord_utils.py

```diff
@@ -0,0 +1,323 @@
+"""Utilites to handle coordinates and transformations."""
+import logging
+import re
+from copy import deepcopy
+from pyproj import Transformer
+import numpy as np
+from .core import Holes, Hole
+
+__all__ = ["project_holes"]
+
+logger = logging.getLogger("pyinfraformat")
+
+TRANSFORMERS = {}  # dict of pyproj Transformers, key (input, output)
```

Comment:
Is this edited somewhere?

Reply:
Yes. Other function add and get transformers. Getting transformer is very time-consuming by Transformer.from_crs(input_epsg, output_epsg). 

    key = (input_epsg, output_epsg)
    if key in TRANSFORMERS:
        transf = TRANSFORMERS[key]
    else:
        transf = Transformer.from_crs(input_epsg, output_epsg)
        TRANSFORMERS[key] = transf

Reply:
Sounds good